dotpay/src/main/java/com/adyen/checkout/dotpay/DotpayRecyclerView.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by arman on 11/6/2019.
+ */
+package com.adyen.checkout.dotpay
+
+import android.content.Context
+import android.util.AttributeSet
+import com.adyen.checkout.components.model.payments.request.DotpayPaymentMethod
+import com.adyen.checkout.issuerlist.IssuerListRecyclerView
+
+class DotpayRecyclerView : IssuerListRecyclerView<DotpayPaymentMethod, DotpayComponent> {
```

Comment:
What is the use of these "RecyclerView"s? It looks like they are all the same, except for `EPSRecyclerView`. I think it would be better to make `IssuerListRecyclerView` a normal class and then in `ComponentParsingProvider` do something like:
```
...
PaymentMethodTypes.EPS -> IssuerListRecyclerView(context).apply { hideIssuersLogo = true }
...
```